src/apps/code-editor/src/app/components/FileActions/components/EditorActions/Sync/Sync.js

```diff
@@ -39,12 +37,11 @@ export const Sync = memo(function Sync(props) {
 
   return (
     <div>
-      <Button type="alt" onClick={() => setOpen(true)}>
-        {loading ? (
-          <FontAwesomeIcon spin icon={faSpinner} />
-        ) : (
-          <FontAwesomeIcon icon={faSync} />
-        )}
+      <Button
```

Comment:
The button this is replacing is an alt button so shouldn't this be 'warning' colored?

Reply:
I added color: 'warning'

@shrunyan How can I test this page?

Reply:
You need to get into a state where your local differs from remote.

- Open browser 1 and edit file A. Do not save.
- Open browser 2 and edit file A. Save.
- Open browser 1 and load file A. See warning.

This is because browser 1 has the unsaved file changes in localstorage and after the file is altered and then the data is refetched it sees the difference and presents the warning.